README.md

```diff
@@ -271,6 +278,98 @@ const result = toTitleCase(str);
 //    ^ 'Hello World'
 ```
 
+## Strongly-typed transformation of objects
```

Comment:
```suggestion
## Strongly-typed shallow transformation of objects
```

Reply:
done in [6cde47a](https://github.com/gustavoguichard/string-ts/pull/16/commits/6cde47a1a68c50d506dc516c25ad05ce56454ed6)